TACC/Core-CMS-Custom
# fix(demdata-cms): build and deploy obstacles

## Overview

Fix final CMS build hurdle i.e. missing `taccsite_custom`.

## Related

- builds upon https://github.com/TACC/Core-CMS-Custom/pull/39

## Changes

- Add placeholder content for `taccsite_custom`.
- ~~Add setting to silence the recaptcha error.~~[^1]
- Add TODO.md docs to permit our shortcuts.

## Testing

1. Build and deploy CMS.
2. Verify no crash and burn.

✅ https://jenkins01.tacc.utexas.edu/job/Core_Portal_Deploy/2111

## UI

Skipped.

[^1]: ~~Ideally, this error is avoided by not loading recaptcha if CMS doe snot use form, but @wesleyboar installed `djangocms-forms` in Core, and it loads recaptcha. Way to go, self; cause future problems.~~ [@fnets moved the setting to pre-prod server, so I can [drop the TODO and warning and setting from the repo](https://github.com/TACC/Core-CMS-Custom/pull/40/commits/5c47afd).]
